### ISSUE TITLE
code about scienceqa

### ISSUE THREAD

**Issue by pkulium** [deepcs233/Visual-CoT#3]
great work! I want to ask where can I find code to evaluate scienceqa?

**Comment by deepcs233** [comment]
Hi!
You can find the script here: https://github.com/deepcs233/Visual-CoT/blob/main/scripts/v1_5/eval/sqa.sh

**Comment by pkulium** [comment]
Thanks for the message! Is this script for visual cot? 
I read it before, but It seems that it is same as the script from llava repo.

**Comment by deepcs233** [comment]
Yes, it's from the llava repo. Our VisCoT benchmark does not contain this dataset.
